zerver/views/home.py

```diff
@@ -173,6 +173,7 @@ def home_real(request: HttpRequest) -> HttpResponse:
     # These end up in a global JavaScript Object named 'page_params'.
     page_params = dict(
         # Server settings.
+        new_user_bot_configured = True if (settings.NEW_USER_BOT is not None) else False,
```

Comment:
You can simplify this to `new_user_bot_configured = settings.NEW_USER_BOT is not None`, since the `is not` check will return either `True` or `False`.

Reply:
Yeah, Done.